qiskit/circuit/quantumcircuit.py

```diff
@@ -3115,6 +3116,13 @@ def decompose(
         # do not copy operations, this is done in the conversion with circuit_to_dag
         return dag_to_circuit(dag, copy_operations=False)
 
+    @deprecate_arg(
+        "justify",
+        since="1.1.0",
+        deprecation_description="Setting QuantumCircuit.draw()’s justify argument to a value other than "
+        "'left', 'right', or 'none'. Until deprecation, default 'left' will be used instead.",
```

Comment:
You could add this:

> Starting in Qiskit 2.0, invalid arguments will error.

Reply:
Done!

Reply:
Actually, the `@deprecate_arg`, already automatically appends this, to the end of the message:
`"is deprecated as of qiskit 1.1.0. It will be removed no earlier than 3 months after the release date."`

So since both, don't fit very well, I'll try to write something else, that looks better with such automazation 👍

Reply:
(repeated comment, since this is also involved)

----
**Update, from call with luciano and Kevin:**

They think in this case the wrapper complicates things instead of helping, so we should maybe remove it 👍